tests/kj-rs/tests.h

```diff
@@ -2,12 +2,16 @@
 
 #include <cstdint>
 #include <kj/async.h>
+#include <kj/debug.h>
+#include "cxx-types.h"
 #include "tests/kj-rs/lib.rs.h"
 
 namespace kj_rs {
 
 kj::Promise<void> c_async_void_fn();
 kj::Promise<int64_t> c_async_int_fn();
 kj::Promise<Shared> c_async_struct_fn();
+kj::Own<CppType> cpp_kj_own();
```

Comment:
these need to be next to CppType.

I actually suggest you fold cxx-types.h in here. There's little to gain in having it separate, this is not some reusable functionality, just a bunch of tests.